### PR TITLE
Header footer

### PR DESCRIPTION
**Pull request by MohammadQHabib**
I made a navigation bar as the main header, drawing from a bootstrap CDN, I also made tabs for all our tentative features and pages. Please let me know if I should make any edits. Also, the footer isn't staying at the bottom of the page, do you have any solutions to that?

### CODE REVIEW

File: views/browse.ejs [hunk 1]
```diff
@@ -1,15 +1,21 @@
 <!DOCTYPE html>
 <html>
+<!-- stock head file that contains all pages metadata-->
 <head>
```
Comment (travisbartholome) [inline views/browse.ejs:4]: If the `<head>` and `</head>` tags are included in head.ejs, I think you can remove them here.

File: views/header.ejs [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<!-- views/header.ejs -->
+<!-- creates inverted (black) navigation bar w/ top that is fixed-->
+<nav class="navbar navbar-inverse navbar-fixed-top">
+    <!-- "container-fludi => takes full width-->
```
Comment (travisbartholome) [inline views/header.ejs]: Nitpick: fix indentation in this file, everything after the first nav tag should be moved back one indent (just highlight it all and shift+tab).

**Review by travisbartholome — COMMENTED** [review]
Can you post a screenshot (in the PR comments) of what this looks like?

A screenshot will also help diagnose the footer problem (I suspect it's just because we don't have CSS telling it to stay at the bottom of the page yet).

File: views/header.ejs (hunk 2 above)
Comment (MohammadQHabib) [reply views/header.ejs]: Yea I did that, for some reason it won't let me commit that as the only change?

Comment (travisbartholome) [reply views/header.ejs]: Weird. Not sure why that would be the case, but worst-case it doesn't really matter that much anyway.

File: views/header.ejs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+<!-- creates inverted (black) navigation bar w/ top that is fixed-->
+<nav class="navbar navbar-inverse navbar-fixed-top">
+<!-- "container-fludi => takes full width-->
+<div class="container-fluid">
+    <div class="navbar-header">
+    <a class="navbar-brand" href="#"><span class="glyphicon glyphicon-sunglasses"></span> Swag Store</a>
+    </div>
+    <ul class="nav navbar-nav">
+    <!-- Add in hyper link to potential About Us page-->
+    <li class="active"><a href="/views/browse.ejs">Browse</a></li>
```
Comment (travisbartholome) [inline views/header.ejs]: Link here should be just `href="/"` - the hrefs need to be the routes we're using in Express, not the names of the view files.

File: views/header.ejs [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    <!-- Add in hyper link to Browse page -->
+    <li><a href="#">About Us</a></li>
+    </ul>
+    <ul class="nav navbar-nav navbar-right">
+    <!-- Add links to checkout and cart pages?-->
+    <li><a href="#"><span class="glyphicon glyphicon-shopping-cart"></span> Cart</a></li>
+    <li><a href="/views/checkout.ejs"><span class="glyphicon glyphicon-log-in"></span> Checkout</a></li>
```
Comment (travisbartholome) [inline views/header.ejs]: Similar to above, link should be `href="/checkout"`

File: views/header.ejs [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    </ul>
+    <form class="navbar-form navbar-right" action="/action_page.php">
```
Comment (travisbartholome) [inline views/header.ejs]: Just get rid of the `action` property on this form element. There's no reason for it here and currently it's calling a page that doesn't exist.

File: views/head.ejs [hunk 6]
```diff
@@ -0,0 +1,10 @@
+<!-- stock head file that contains all pages metadata-->
+    <title>COSC4351 Bartholome-Le-Habib Ecommerce Store</title>
```
Comment (travisbartholome) [inline views/head.ejs]: Can we get rid of the indent here?

**Comment by travisbartholome** [comment]
image

Putting this screenshot here for reference. I think this looks pretty good in terms of the header/footer content @MohammadQHabib (minus the weird footer placement, but we can fix that later when we have a full page of items). If you want to merge after fixing the comments I made above, let's do it.

**Pull request merged**